scripts/load-people

```diff
@@ -146,20 +150,25 @@ sub load_people {
     foreach (@{$j->{posts}}) {
         $posts{$_->{id}} = $_;
     }
+    $start_transaction->execute();
     foreach (@{$j->{memberships}}) {
         if ($_->{redirect}) {
             $gradd->execute($_->{id}, $_->{redirect});
         } else {
             load_member($_);
         }
     }
+    $query_commit->execute();
```

Comment:
Not sure I follow this logic - if we're putting this in a transaction, don't we want all of memberships and people to be committed at once? Otherwise you could have memberships in the DB that talk about a person that's not yet been inserted.

Reply:
Re grouped it so it was doing both in the same commit.